platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/event/tipset/TipsetEventCreatorTests.java

```diff
@@ -1025,7 +1025,7 @@ void noStaleEventsAtCreationTimeTest(final boolean useBirthRoundForAncient) {
     /**
      * Checks that birth round on events is being set if the setting for using birth round is set.
      * <p>
-     * FUTURE WORK: Update this test to use RosterDiff instead of EventWindow
+     * FUTURE WORK: Update this test to use AddressBookDiff instead of EventWindow
```

Comment:
this comment change should probably be reverted or updated to something different.